example/zend-3.0.3/Vagrantfile

```diff
@@ -0,0 +1,56 @@
+# -*- mode: ruby -*-
```

Comment:
Similar to the dockerfile, if the example can be run with Vagrant we should mention it in the Readme